src/proto/mgmt/storage_query.proto

```diff
@@ -74,3 +74,13 @@ message SmdPoolResp {
 	int32 status = 1;
 	repeated Pool pools = 2;
 }
+
+message DevStateReq {
+	string dev_uuid = 1; // UUID of blobstore
+}
+
+message DevStateResp {
+	int32 status = 1; // DAOS error code
+	string dev_uuid = 2; // UUID of blobstore
```

Comment:
why do we need to return a uuid, Will it ever be different to that requested?

Reply:
Just to be able to print the dev uuid to console with the status again for clarity, might be redundant? not opposed to taking it out

Reply:
you could use the value in the request to print to the console but it's not a big issue